spec/views/hq/sponsors/index_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'will_paginate/array'
```

Comment:
I think the alternative to this is to replace `assign(:sponsors, [])` with `assign(:sponsors, Sponsor.none)` which is actually closer to the conditions the actual code will encounter, as it is an empty ActiveRecord::Relation enumerable instead of an Array